fast64_internal/oot/data/xml/ActorList.xml

```diff
@@ -931,12 +931,12 @@ ZROT 0x00FF = Gold Skulltula spawn var high byte (See Gold Skulltula Actor 0095)
       <Item Params="0010"> Line Loop</Item>
       <Item Params="0020"> Circle Loop</Item>
     </Type>
-    <Property Index="1" Mask="0xF000" Name="Radius" TiedActorTypes="0000,0020"/>
-    <Property Index="2" Mask="0x0001" Name="Move North"/>
-    <Property Index="3" Mask="0x0002" Name="Move South"/>
-    <Property Index="4" Mask="0x0004" Name="Move West"/>
-    <Property Index="5" Mask="0x0008" Name="Move East"/>
-    <Property Index="6" Mask="0x0F00" Name="Attack Speed"/>
+    <Property Index="1" Mask="0xF000" Name="Speed" TiedActorTypes="0000,0020"/>
+    <Property Index="6" Mask="0x0F00" Name="Radius"/>
+    <Bool Index="2" Mask="0x0001" Name="Move North"/>
+    <Bool Index="3" Mask="0x0002" Name="Move South"/>
+    <Bool Index="4" Mask="0x0004" Name="Move West"/>
+    <Bool Index="5" Mask="0x0008" Name="Move East"/>
```

Comment:
iirc those indices need to be in numerical order, I think it's fine but it should be from 1 to whatever per param type (so here it's ``<Bool>``), here it starts at 2

also the 2nd ``<Property>`` index should be 2 not 6